envoy/pkg/probe/cmd/root.go

```diff
@@ -74,7 +184,12 @@ var rootCmd = &cobra.Command{
 			httpsAddr: config.httpsAddr,
 			timeout:   config.timeout,
 		}
+		lm := &livenessMonitor{m}
+		rm := &readinessMonitor{m}
+
 		mux.Handle("/", m)
+		mux.Handle("/healthz", lm)
+		mux.Handle("/ready", rm)
```

Comment:
Is there a reason for the presence or absence of `z`?
It would be better to unify the naming convention.

K8s components seems to using the `readyz` as the health-check endpont.
https://kubernetes.io/docs/reference/using-api/health-checks/

Reply:
I've fixed :) (And also REAME.md)